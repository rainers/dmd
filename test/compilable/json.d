// PERMUTE_ARGS:
// REQUIRED_ARGS: -o- -X -Xftest_results/compilable/json.out
// POST_SCRIPT: compilable/extra-files/json-postscript.sh

struct X;

enum Y;

// 3404
alias int myInt;
myInt x;

// 3466
struct Foo3466(T) { T t; }
class  Bar3466(T) { T t; }

// 4178

struct  Bar4178 {
  this(int i)  { }
  this(this) { }
  ~this() { }
}


