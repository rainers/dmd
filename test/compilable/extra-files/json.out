[
{
"kind" : "module",
"members" : [
{
"name" : "X",
"kind" : "struct",
"protection" : "public",
"line" : 5}
,
{
"name" : "Y",
"kind" : "enum",
"protection" : "public",
"line" : 7}
]
}
]
